Goal: Document that chpasswd foes not use PAM.
Fixes: #396726

Status wrt upstream: Not reported yet.

Note: Even when PAM support is activated in chpasswd, this patch is valid. The
PAM support is only for the authentication of the caller.

Index: shadow-4.0.18.1/man/chpasswd.8.xml
===================================================================
--- shadow-4.0.18.1.orig/man/chpasswd.8.xml	2007-06-19 11:43:16.000000000 +0100
+++ shadow-4.0.18.1/man/chpasswd.8.xml	2007-06-19 11:47:10.000000000 +0100
@@ -79,6 +79,13 @@
       Remember to set permissions or umask to prevent readability of
       unencrypted files by other users.
     </para>
+    <para>
+      <command>chpasswd</command> does not use PAM to update the passwords.
+      Thus, It only updates the passwords from the
+      <filename>/etc/passwd</filename> and <filename>/etc/shadow</filename>
+      databases, and does not support the various checks provided by PAM
+      modules.
+    </para>
   </refsect1>
 
   <refsect1 id='see_also'>
